tests/test_flobject.py

```diff
@@ -653,6 +653,7 @@ class a_2(Boolean):
     )  # noqa: W293
 
 
+@pytest.mark.nightly
 def test_accessor_methods_on_settings_object(load_static_mixer_case):
```

Comment:
@mkundu 
Tests like this are heavily exercising the API, that's true, but here we most interested interested in the behaviour of the client-side flobject.py. In fact, for CI, we want to check that we haven't broken flobject.py, and we can do that with a stable API version. So, shall we mark test such as this one as using the latest released fluent version, currently 23R2? In terms of marking tests, it would be useful to have labels to indicate _latest released version_ and _current development version_. Then those labels can be maintained and one place and incremented as Fluent is released. I don't believe we have something like that(?)

Reply:
`pytest.mark.fluent_version("latest")` means it will run

1. with latest dev version of Fluent in PRs, and
2. with latest release and dev versions of Fluent in nightly